tests/profile/profile-page.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test } from '@playwright/test';
+
+import { ProfilePage } from '../page-objects/profile-page';
+
+let profilePage: ProfilePage;
+
+test.describe('Profile Page - Basic display tests', () => {
+  test.describe.configure({ mode: 'serial' });
+
+  test(`Profiles use profile page layout`, async ({ browser }) => {
+    const browserPage = await browser.newPage();
+    profilePage = new ProfilePage(browserPage);
+
+    await test.step(`Go to @brewster's profile`, async () => {
+      await profilePage.visit('brewster');
+    });
+
+    await test.step(`Check if profile avatar, summary and action bar appear`, async () => {
+      await profilePage.validatePageHeaderElements();
+    });
+
+    await test.step(`Check if profile tabs are displayed`, async () => {
+      await profilePage.validateUnownedProfilePageTabs();
+    });
```

Comment:
You can also add validateOwnProfilePageTabs() below to test own profile page tab in addition of other tabs.

Reply:
I think actually running the owner tests would require a separate logged-in setup (these are currently run as a guest account IIUC). I think setting up the logged-in user might be another ticket?

Reply:
> I think setting up the logged-in user might be another ticket?
yes that's correct... we can work on logged-in setup once this [PR](https://github.com/internetarchive/archiveorg-e2e-playwright/pull/33) is merged.

with that, can we specifically update the test description to say it's guest validating as guest account?

Reply:
Yes, the logged-in case is in-review from @dualcnhq, once she merged in her changes for auth setup, we can work on loans tab test later.